_data/applications.yml

```diff
@@ -176,6 +176,15 @@ navigator:
   prerelease: false
   description: |
     A Featureful File Browser for Cockpit.
+
+identities:
+   title: Cockpit Identities
+   source: https://github.com/45drives/cockpit-identities
+   issues: https://github.com/45Drives/cockpit-identities/issues
+   official: false
+   prerelease: false
+   description: |
+     A Cockpit plugin to easily manage users and groups along with associated items.
```

Comment:
Can you please make the description a bit more concrete? The builtin Accounts page can do all the [listed features](https://github.com/45drives/cockpit-identities?tab=readme-ov-file#features) except for setting the Samba password. I.e. please drop "easily", if possible avoid "manage", and replace the vague "associated items" with something more specific. I.e. it should answer the question "why and when would I want to install this?" Unfortunately the upstream project itself isn't very verbose in that regard.

Reply:
To be more specific -- the 45drives apps are not very reliable. They have zero tests, do, well, "interesting" things with shell commands, don't fit the general Cockpit UI look and guidelines, etc. Presumably they did these for a specific product, but Cockpit is supported across a wide range of operating systems.

In other words, we don't want to recommend this page for standard Unix user management. If there's some "killer feature" which the Accounts page doesn't have, it should be pointed out as such.